### PR TITLE
thing-of-the-day: Fix minor bug

### PR DESCRIPTION
**Pull request by Distrib-ps**
Pr for https://www.smogon.com/forums/threads/bug-reports-v4-read-original-post-before-posting.3663703/post-8710248

### CODE REVIEW

**Comment by mia-pi-git** [comment]
Why toID this? Why not just escape the HTML?

File: server/chat-plugins/thing-of-the-day.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ class OtdHandler {
 
 		const nomObj = {
 			nomination: nomination,
-			name: user.name,
+			name: toID(user),
```
Comment (AnnikaCodes) [inline server/chat-plugins/thing-of-the-day.ts:190]: This shouldn't be a user ID - you should just escape it with `Utils.escapeHTML` (either here or when it's displayed).

**Review by AnnikaCodes — CHANGES_REQUESTED** [review]

File: server/chat-plugins/thing-of-the-day.ts [hunk 2]
```diff
@@ -230,7 +230,7 @@ class OtdHandler {
 		const entries = [];
 
 		for (const value of this.nominations.values()) {
-			entries.push(`<li><b>${value.nomination}</b> <i>(Submitted by ${value.name})</i></li>`);
+			entries.push(`<li><b>${value.nomination}</b> <i>(Submitted by ${Utils.escapeHTML(value.name)})</i></li>`);
```
Comment (AnnikaCodes) [inline server/chat-plugins/thing-of-the-day.ts:233]:
```suggestion
			entries.push(Utils.html`<li><b>${value.nomination}</b> <i>(Submitted by ${value.name})</i></li>`);
```
IIRC we prefer this style.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**